ping/ping_json.c

```diff
@@ -0,0 +1,217 @@
+/*
+ * SPDX-License-Identifier: BSD-3-Clause
+ * Copyright (c) 2024 Georg Pfuetzenreuter <mail+ip@georg-pfuetzenreuter.net>
+ */
+
+#include "ping.h"
+#include "stdarg.h"
+
+static inline void json_emergency(char *msg) {
+	error(1, 0, "%s", msg);
+}
+
+static inline void test_buflen(int have) {
+	if (have < 0 || have >= PING_JSON_MAX)
+		json_emergency("Overflow during JSON construction.");
```

Comment:
nit: Although I'm not a fan of iputils being translated, when we translate everything, this should be also translated (with `_(...)`). Also please no dot (`.`) at the end of the message.

Reply:
Sure!